README.md

```diff
@@ -302,9 +302,9 @@ Please make sure you run the tests locally before submitting your PR (see [Runni
 
 #### Employees: Triggering CI for a contributed PR
 
-Use branch `master` and add `BUILDKITE_REFSPEC="refs/pull/${PR_NUM}/head"` and `BUILDKITE_COMMIT=${SHA}` in the Environment Variables section.
+Use branch `external_contrib_ci` and the specific sha of the commit you want to build. Add `BUILDKITE_REFSPEC="refs/pull/${PR_NUM}/head"` in the Environment Variables section.
```

Comment:
Can we link it here?

Reply:
Link what?

Reply:
The external build pipeline.

Reply:
Oh! It's linked in the para above, but you're right that it's more relevant here. I'll move it.